### PR TITLE
3 changes/fixes around how tuples are output by the replayer.

### PR DESCRIPTION
**Pull request by gregschohn**
Simplify and fix the json output of the 'tuples' that the replayer emits.  No longer wrap the tuples within a log4j json record, just shoot it straight out, followed by a newline.  Furthermore, the bodies often had many null values within them.  That's been fixed by copying just the readable bytes rather than taking the whole byte array.

### Description

* Category Bug fix
* Why these changes are required? Looking at the output was just about impossible
* What is the old behavior before changes and new behavior after changes? Now users can run humanReadableLogs.py and look at the output that it dumps to readable-tuples.log.  Users will also find that it is easier to read the tuple output even without that.

### Issues Resolved
https://github.com/opensearch-project/opensearch-migrations/issues/511
https://opensearch.atlassian.net/browse/MIGRATIONS-1510

### Testing
Updated unit tests.  Ran the docker solution and a standalone replayer with an input stream.

### Check List
- [ ] New functionality includes testing
  - [x] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by lewijacn — APPROVED** [review]
Very happy to see this fix

File: TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/ResultsToLogsConsumerTest.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public void testOutputterForGet() throws IOException {
                         "        \"Status-Code\": 200,\n" +
                         "        \"Reason-Phrase\": \"OK\",\n" +
                         "        \"response_time_ms\": 0,\n" +
-                        "        \"body\": \"SFRUUC8xLjEgMjAwIE9LDQpDb250ZW50LXRyYW5zZmVyLWVuY29kaW5nOiBjaHVua2VkDQpEYXRlOiBUaHUsIDA4IEp1biAyMDIzIDIzOjA2OjIzIEdNVA0KVHJhbnNmZXItZW5jb2Rpbmc6IGNodW5rZWQNCkNvbnRlbnQtdHlwZTogdGV4dC9wbGFpbg0KRnVudGltZTogY2hlY2tJdCENCg0KMWUNCkkgc2hvdWxkIGJlIGRlY3J5cHRlZCB0ZXN0ZXIhCg0KMA0KDQo=\",\n" +
+                        "        \"body\": \"SSBzaG91bGQgYmUgZGVjcnlwdGVkIHRlc3RlciEK\",\n" +
```
Comment (sumobrian) [inline TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/ResultsToLogsConsumerTest.java:159]: I was hoping there was funny message encoded ◡̈

Comment (sumobrian) [inline TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/ResultsToLogsConsumerTest.java:159]: I was hoping there was funny message encoded ◡̈

**Review by sumobrian — APPROVED** [review]

**Comment by sumobrian** [comment]
Was human readable script tested?

**Comment by gregschohn** [comment]
Yes, the script was tested against a simple run. There still aren't unit tests, so I'm not sure exactly what the behavior should be.

**Pull request merged**